test/extended/cli/idle.go

```diff
@@ -0,0 +1,148 @@
+package cli
+
+import (
+	"fmt"
+	g "github.com/onsi/ginkgo"
```

Comment:
Nit: I like imports in 3 separate groups:
1. builtins
2. external
3. openshift 

:wink: